RijsMuseum/app/src/main/res/layout/activity_main.xml

```diff
@@ -16,7 +16,10 @@
         app:layout_constraintBottom_toBottomOf="parent"
```

Comment:
you no longer need `android:name="com.example.rijsmuseum.fragments.ArtefactsFragment"` because the navigation graph will take care of setting the fragments